dom0/sd-dom0-files.sls

```diff
@@ -60,12 +81,19 @@ dom0-remove-securedrop-workstation-stretch-template:
       - file: dom0-workstation-rpm-repo
 
 dom0-install-securedrop-workstation-template:
+{% if grains['osrelease'] == '4.1' %}
+  qvm.template_installed:
+    - name: securedrop-workstation-buster
+{% else %}
```

Comment:
Nit: Would it maybe be better to use`{% elif  grains['osrelease'] == '4.0' %}` here? I'm sure we'll be removing this conditional logic pretty soon, but it seems best to be always explicit about the version-based branching.